.github/workflows/tag-latest.yml

```diff
@@ -14,7 +14,8 @@ on:
       build-id:
         description: The build-id (or any tag) that you want to tag as 'latest'
         required: true
-
+  schedule:
+    - cron: '0 0 * * MON'
```

Comment:
Flagging that

1. I don't actually know when the images are going to be "up"
2. You'll need to make the thing smart enough to fill in it's own build-id

For problem 1 you can probably solve this by just delaying an hour. Problem two is harder...

Reply:
Yes but it looks like a timestamp so you've got variance

![image](https://github.com/UWIT-IAM/docker-library/assets/26555712/94dbd72a-2647-406a-9fce-ab5e47dc9c09)

note how all the the minute/seconds vary slightly

Reply:
I may have led you astray. It might be easier to just add "latest" to the other workflow that's currently building the images

- [poetry](https://github.com/UWIT-IAM/docker-library/blob/main/.github/workflows/poetry-update.yml#L44)
- [uw-saml-poetry](https://github.com/UWIT-IAM/docker-library/blob/main/.github/workflows/poetry-update.yml#L56)

Reply:
@miker985 
so, for example, in the poetry image...
We would go from 
`dest_image: ghcr.io/uwit-iam/poetry:${{ steps.configure.outputs.build-id }}` to
`dest_image: ghcr.io/uwit-iam/poetry:${{ steps.configure.outputs.build-id }}:latest`?

Reply:
No that would be syntatically invalid.

You could do `dest_image: ghcr.io/uwit-iam/poetry:latest` **but** then you'd have a hard time getting any image except the "latest".

Instead you want to update [these lines](https://github.com/UWIT-IAM/docker-library/blob/main/.github/workflows/poetry-update.yml#L39-L56) and add 4 new ones

```yaml
      - run: |
          docker build -f images/poetry/Dockerfile -t ${{ env.dest_image }} images/poetry
          docker push ${{ env.dest_image }}
          docker push ${{ env.latest_image }}  # <== NEW
          echo ::notice::Pushed ${{ env.dest_image }}
        env:
          dest_image: ghcr.io/uwit-iam/poetry:${{ steps.configure.outputs.build-id }}
          latest_image: ghcr.io/uwit-iam/poetry:latest  # <== NEW
        name: Re-build poetry base image
      - run: |
          set -x
          docker build -f images/uw-saml-poetry/Dockerfile \
            --build-arg from_base_tag=${{ env.build_id }} \
            -t ${{ env.dest_image }} images/uw-saml-poetry
          docker push ${{ env.dest_image }}
          docker push ${{ env.latest_image }}  # <== NEW
          echo ::notice::Pushed ${{ env.dest_image }}
        name: Re-build uw-saml-poetry image
        env:
          build_id: ${{ steps.configure.outputs.build-id }}
          dest_image: ghcr.io/uwit-iam/uw-saml-poetry:${{ steps.configure.outputs.build-id }}
          latest_image: ghcr.io/uwit-iam/uw-saml-poetry:latest  # <== NEW
```

Reply:
what's the update? or, is the update the new lines?

Reply:
I've updated the code section to include 2 changed lines that I somehow failed to correctly copy/paste in.

During `docker build` you provide `-t` twice (once for the `build-id` tag and once for the `latest` tag). Then you push both of those tagged images

Reply:
Thank you, Mike!